src/main/java/com/base/site/security/DbInitializer.java

```diff
@@ -28,11 +33,14 @@ public void run(String... args) throws Exception {
         /*
         log.info("Inside the DbInitializer");
         //usersRepo.deleteAll();
+
+        UserType userType = userTypeRepo.getById(1L);
+
         List<Users> userList = new ArrayList<>();
-        userList.add(new Users("User1", "user1", "user@user.dk", passwordEncoder.encode("user1"),1,"USER",1));
-        userList.add(new Users("User2", "user2", "user2@user2.dk", passwordEncoder.encode("user2"),1,"USER",1));
-        userList.add(new Users("admin1", "admin1", "admin1@admin1.dk", passwordEncoder.encode("admin1"),1,"ADMIN",1));
-        userList.add(new Users("admin2", "admin2", "admin2@admin2.dk", passwordEncoder.encode("admin2"),1,"",1));
+        userList.add(new Users("User1", "user1", "user@user.dk", passwordEncoder.encode("user1"),userType,"USER",1));
+        userList.add(new Users("User2", "user2", "user2@user2.dk", passwordEncoder.encode("user2"),userType,"USER",1));
+        userList.add(new Users("admin1", "admin1", "admin1@admin1.dk", passwordEncoder.encode("admin1"),userType,"ADMIN",1));
+        userList.add(new Users("admin2", "admin2", "admin2@admin2.dk", passwordEncoder.encode("admin2"),userType,"",1));
 
```

Comment:
these lines needs to be commented out otherwise we might create these users evertime we log in.

Reply:
I think they are already commented out